portfolio/src/main/webapp/tasksApi.js

```diff
@@ -1,64 +1,16 @@
-var GoogleAuth;
+var importMenuVisible;
```

Comment:
I think we should not keep track in Javascript like this. I think we should keep track of the state in the DOM. For example, it can be some attribute in the toggle tasks button, like data-is-import-menu-visible="true".

Reply:
Done.